frontend/packages/console-app/src/components/guided-tours/utils/guided-tour-utils.ts

```diff
@@ -15,3 +16,14 @@ export const getGuidedToursWithStatus = (): GuidedTourCatalogItem[] => {
     };
   });
 };
+
+export const getGuidedTour = (tourID: string): GuidedTourItem => {
+  return tourID && mockGuidedTours.find((tour) => tour.id === tourID);
+};
+
+export const setTourStatus = (tourID: string, tourStatus: GuidedTourStatus) => {
+  tourID && localStorage.setItem(tourID, tourStatus);
+};
+
+export const getTourStatus = (tourID: string): GuidedTourStatus =>
+  tourID && (localStorage.getItem(tourID) as GuidedTourStatus);
```

Comment:
Remove.
Saving to and retrieving from local storage is beyond the scope of this story.

Reply:
done